source/module_base/module_container/ATen/core/tensor_buffer.cpp

```diff
@@ -22,6 +23,7 @@ TensorBuffer::TensorBuffer(base::core::Allocator* alloc, size_t size) {
     alloc_ = alloc; 
     if (size > 0) {
         data_ = alloc_->allocate(size);
+        ModuleBase::Memory::record("TensorBuffer::data_",sizeof(size_t)*size);
```

Comment:
`Tensor` is a container, it will used by developers for different purpose, the memory should be recorded by key word : `TensorBuffer::data_`.
I don't agree to record memory here, you should find the actual instantiated variables.